content/en/synthetics/guide/email-validation.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Email Validation with Browser Tests
+kind: documentation
+description: Verify an email and its content with browser test steps
+further_reading:
+- link: "//synthetics/browser_tests/actions"
+  tag: "Documentation"
+  text: "Learn about steps for browser tests"
+- link: "/synthetics/browser_tests/advanced_options/"
+  tag: "Documentation"
+  text: "Configure advanced options for steps"
+---
+
+## Overview
```

Comment:
Can we add an `Overview` section to the other guides for homogeneity or remove it from that one?